schema/gendummydata.py

```diff
@@ -18,6 +18,18 @@
 
 import bcrypt
```

Comment:
this actually should not be done with system, imho people should use a dedicated python3 virtual environment while generating the dummy data for it. Which they could do, also, without bcrypt it will not run and complain about missing deps which should not be needed to be covered actually. Since its only developer related thing I donot think we should have additional try catches unnecessarily.